pkg/openstack/actuator.go

```diff
@@ -129,6 +140,34 @@ func (a *OpenStackActuator) sync(ctx context.Context, cr *minterv1.CredentialsRe
 	return nil
 }
 
+// fixInvalidCACertFile insures that clouds.yaml has the right CACertFile value
+// For more information: https://bugzilla.redhat.com/show_bug.cgi?id=1940142
+func (a *OpenStackActuator) fixInvalidCACertFile(content string) (string, error) {
+	var clouds clientconfig.Clouds
```

Comment:
Not my code, but `gophercloud` seems like a heavy dependency for massaging some YAML.  Is the format of this content so complicated that it's worth adding the dependency instead of inlining enough knowledge locally to make the required pivot?

Reply:
That's a good point. I've pushed a modified version of this patch here which doesn't use gophercloud: https://github.com/mdbooth/cloud-credential-operator/tree/correct_cacert

I'll discuss with @Fedosin if he wants to use it.